app/uk/gov/hmrc/cardpaymentfrontend/controllers/PaymentFailedController.scala

```diff
@@ -53,7 +53,7 @@ class PaymentFailedController @Inject() (
         (formWithErrors: Form[ChooseAPaymentMethodForm]) => BadRequest(paymentFailedPage(taxType = "Self Assessment", true, formWithErrors)),
```

Comment:
This is not quite right also, it should not be hard coded to self assessment. We want to update this to be generic so that we can onboard the other regimes.